docker/start-bootnode.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/sh -x
```

Comment:
need to change to bash as you use string compare
https://stackoverflow.com/questions/12230690/string-comparison-in-bash-not-found